src/main/java/de/tum/in/www1/artemis/web/rest/ModelingSubmissionResource.java

```diff
@@ -81,7 +82,7 @@ public ModelingSubmissionResource(JsonAssessmentRepository jsonAssessmentReposit
      */
```

Comment:
The comment is not very detailed. Is this the save Submission button? What happens if this method is called after the due date of an exercise? Do you handle this case?